bitnami/etcd/templates/svc.yaml

```diff
@@ -10,6 +10,12 @@ spec:
   type: {{ .Values.service.type }}
   {{- if and (eq .Values.service.type "LoadBalancer") (not (empty .Values.service.loadBalancerIP)) }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+  {{- end }}
+  {{- if and (eq .Values.service.type "LoadBalancer") (not (empty .Values.service.loadBalancerSourceRanges)) }}
+  loadBalancerSourceRanges:
+  {{- with .Values.service.loadBalancerSourceRanges }}
```

Comment:
Is the `with` instruction necessary? wouldn't using toYaml with the array be enough?

Reply:
Thank for the feedback. You are right. But I saw some of chart use this pattern, some of not.
use `with`:
https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/redis-master-svc.yaml#L23-L28
https://github.com/bitnami/charts/blob/master/bitnami/influxdb/templates/influxdb/service.yaml#L16-L21
just use `toYaml`:
https://github.com/bitnami/charts/blob/master/bitnami/external-dns/templates/service.yaml#L19-L21
https://github.com/bitnami/charts/blob/master/bitnami/thanos/templates/compactor/service.yaml#L19-L21

Reply:
Yes, we are trying to normalize it Could you go for toYaml instead?

Reply:
sure